README.md

```diff
@@ -43,7 +43,62 @@ There are two types of workflows you can use to contribute documentation:
+   > ✔️ Once your PR is merged, your documentation is online.
 
 ### Contributing via Github<a id="Github"></a>
 
+Writing or editing files directly on Github is a good way to get started if you haven't worked with the command line or IDEs or just want to make a quick change. 
+
+#### Creating a new branch:
+
+1. Pick an issue you want to work on.
+2. Use _Create a branch_ to create a new branch that is linked to the issue. <br/>
+   <img width="252" alt="image" src="https://user-images.githubusercontent.com/13576110/172503283-338f72a5-29c4-4131-b159-040a35c5cffe.png">
+3. Change the branch source to `docs` and name the branch how you prefer. The default is the number and name of the issue.<br/>
+   <img width="500" alt="branch" src="https://user-images.githubusercontent.com/13576110/172506197-58ffe5c5-d356-444b-aa6d-a9629897350e.png">
+   > ℹ️ After you click _Create branch_, a new window pops up with information that you can use in your local development setup. Since you want to work on the website, just dismiss this window. 
+4. Click on the branch to switch to you working branch. <br/>
+   <img width="332" alt="image" src="https://user-images.githubusercontent.com/13576110/172505635-c3197f8c-eef2-42a5-9297-766e34e85b81.png">
+
+
+#### To edit a file in Github: 
+
+1. In your working branch, locate the file you want to edit. 
+2. Verify that you are in your working branch. <br/>
+   <img width="600" alt="branch-overview" src="https://user-images.githubusercontent.com/13576110/172507864-b76f3163-f084-48cc-831e-233779c76934.png">
+3. Hit the _Edit this file_ button.<br/>
+   <img width="429" alt="image" src="https://user-images.githubusercontent.com/13576110/172506536-94669369-a5a3-4b2a-9aa5-66eeeebf2442.png">
+4. Edit the file using standard Markdown syntax. To see a preview, switch to the _Preview_ tab. <br/>
+   <img width="291" alt="image" src="https://user-images.githubusercontent.com/13576110/172506797-4cb454e4-8c4c-4db9-85fc-7ebe9568c0c9.png">
+5. Supplement a meaningful commit message. Remember: We are open source and your changes are visible to the public. You can commit directly on your working branch.<br/>
+   <img width="738" alt="image" src="https://user-images.githubusercontent.com/13576110/172507525-a51253e3-0bc1-41c3-9c57-1ae196af92b0.png">
+   > ✔️ Congrats, you just created a **commit**.
+
+#### Creating a new pull request (PR): 
+1. Navigate to the tree view of your working branch. The URL should look like this: 
+   ```
+   https://github.com/thegooddocsproject/chronologue/tree/your-branch-name
+   ```
+2. Click on _Contribute_ and use the _Compare_ option. <br/> 
+   <img width="726" alt="Go to file" src="https://user-images.githubusercontent.com/13576110/172509325-698203d3-76e9-4065-a481-df432ae01113.png">
+3. Because we want to merge our changes in the `docs` branch, make sure that your changes are compared `docs` and **not** `main`. <br/>
+   Click on _Create pull request_. 
```

Comment:
Google Developer's Style Guide also recommends simply using "click" and not "click on." https://developers.google.com/style/word-list#letter-c